rfcs/0180-broken-package-removal.md

```diff
@@ -0,0 +1,94 @@
+---
+feature: broken-package-removal
+start-date: 2024-07-14
+author: Jörg "Mic92" Thalheim
+co-authors:
+shepherd-team: @emilazy @preisi @jopejoe1
+shepherd-leader: @jopejoe1
+related-issues:
+---
+
+# Summary
+
+[summary]: #summary
+
+This RFC defines conditions under which we remove broken packages or unmaintained leaf packages.
```

Comment:
It says leaf packages here but later goes on to talk about dependant packages which is contradictory. I assume this was once started for leaf packages only but later widened in scope but you forgot to update the summary?

```suggestion
This RFC defines conditions under which we remove broken packages or unmaintained packages.
```

Reply:
Broken packages with reverse dependencies will be removed. Unmaintained packages with reverse dependencies will not. Perhaps the wording could be clarified.

Reply:
I guess people are interpreting the wording later on in the RFC as not referring solely to unmaintained leaf package, but I can’t speak for @Atemu :)

Reply:
As explained in my initial comments first sentence, that's indeed the source of my confusion. You would not need to handle reverse dependencies if it were only about leaf packages because leaf packages do not have reverse dependencies by definition.

Reply:
Reading the RFC again, the later section only mentions removal of dependent packages of *broken* packages, not unmaintained packages. That's the source of my confusion.